faust/assignor/partition_assignor.py

```diff
@@ -327,7 +327,10 @@ def _global_table_standby_assignments(
                         assignment.actives.get(changelog_topic_name, [])
                     )
                     # Only add those partitions as standby which aren't active
-                    standby_partitions = all_partitions - active_partitions
+                    if not table.is_global_global:
+                        standby_partitions = all_partitions - active_partitions
+                    else:
+                        standby_partitions = all_partitions
                     assignment.standbys[changelog_topic_name] = list(standby_partitions)
                     # We add all_partitions as active so they are recovered
                     # in the beginning.
```

Comment:
And as you see here anyway all partitions are already in the actives for global tables already because only actives are fully recovered before the app starts to work.